### ISSUE TITLE
Mismatch between fault block and lithology block next to faults

### ISSUE THREAD

**Issue by pytzcarraldo** [cgre-aachen/gempy#79]
Sometimes there's apparently a mismatch between the fault block and the lithology block near the fault surface. Probably related to the new gradient-based approach and the discretization.

![capture](https://user-images.githubusercontent.com/17277800/41720456-fd80629c-755a-11e8-89fc-fb9da5a59f87.PNG)

Which can mess with topology:

![capture2](https://user-images.githubusercontent.com/17277800/41720474-0ec5e3c4-755b-11e8-9278-746d1ef5ce61.PNG)

**Comment by fastamo** [comment]
How do I give this a +1? This issues is messing with my approach :(

**Comment by fastamo** [comment]
So I think I solved the issue by adding more points to the fault surface. Before, I had all points in a "line", dipping parallel to the created fault surface. Now I added two lateral points, to include the strike-direction of the fault. Thus, I have 4 endpoints (can be imagined of the fault ellipse in a finite fault) and a few in the center. The mismatch is completely gone for me. 

We will probably need to add an assert to make sure there are enough points defined to create a reliable fault/lith block.
